src/assets/main.css

```diff
@@ -3,6 +3,7 @@
 :root {
   --color-dark: #333333;
   --color-light: #DAC389;
+  --btn-width: 24px;
```

Comment:
not necessary to create a variable here. you can just do it ones in the style of the component itself